src/models/board.ts

```diff
@@ -1,15 +1,15 @@
-import Sequelize = require('sequelize');
+const board = (sequelize, Sequelize) => {
+  return sequelize.define('board', {
+    id: {
+      type: Sequelize.UUID,
+      defaultValue: Sequelize.UUIDV4,
+      primaryKey: true,
+    },
+    name: Sequelize.STRING(255),
+    timestamps: true,
+    createdAt: 'date_created',
+    updatedAt: 'date_updated',
+  });
+};
 
-// const Board = Sequelize.define('board', {
-//   id: {
-//     type: Sequelize.UUID,
-//     defaultValue: Sequelize.UUIDV4,
-//     primaryKey: true,
-//   },
-//   name: Sequelize.STRING(255),
-//   timestamps: true,
-//   createdAt: 'date_created',
-//   updatedAt: 'date_updated',
-// });
-
-// module.exports = Board;
+export = board;
```

Comment:
Should use proper syntax. See [TypeScript Deep Dive](https://basarat.gitbooks.io/typescript/content/docs/project/external-modules.html) for details.

Reply:
We opted out of "proper syntax' already when we declined inline `exports`.

Reply:
[This syntax is supposed to be paired with a different import syntax.](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require) Let's stick with `export { thing };` for all our exports as it only deviates from the style of the docs.